v2/brigadier/src/jobs.ts

```diff
@@ -1,15 +1,56 @@
 import { Event } from "./events"
+import { ConcurrentGroup, SerialGroup } from "./groups"
+import { Runnable } from "./runnables"
 
 const defaultTimeout: number = 1000 * 60 * 15
 
-export class Job {
+/**
+ * A Brigade job.
+ * 
+ * Instances of Job represent containers that your Brigade script can
+ * run using the Job#run method.
+ * 
+ * The Job#primaryContainer, initialised from the constructor image argument, determines
```

Comment:
Nit: We've used American English spellings throughout the project, so "initialized" would be more consistent.

Non-blocking.

Reply:
Oops! Thanks for catching this!

Reply:
btw... purely about consistency and not anything else.